0001-typedb/01-start/02-installation.md

```diff
@@ -0,0 +1,328 @@
+
+   <!-- test-ignore -->
+   ```bash
+   sudo apt install openjdk-11-jre
+   ```
+
+   TypeDB supports the [OpenJDK](https://jdk.java.net) and
+   [Oracle JDK](https://www.oracle.com/java/technologies/downloads/).
+
+4. Check the latest version number for typedb-server and its dependencies:
+
+   <!-- test-ignore -->
+   ```bash
+   apt show typedb-server
+   ```
+
+   This will show you console output like this:
+
+   <!-- test-ignore -->
+   ```bash
+   Package: typedb-server
+   Version: 2.15.0
+   Priority: optional
+   Section: contrib/devel
+   Maintainer: Vaticle <community@vaticle.com>
+   Installed-Size: unknown
+   Depends: openjdk-11-jre, typedb-bin (=2.12.0)
+   Download-Size: 71.8 MB
+   APT-Sources: https://repo.vaticle.com/repository/apt trusty/main all Packages
+   Description: TypeDB (server)
+   ```
+
+   Take a note of the latest typedb-server version shown at the `Package` field and the correspondent `typedb-bin` 
+   package version shown at the `Depends` field.
+
+   If we want some other version of TypeDB installed, we can use the `apt show typedb-server=2.14.1` where `2.14.1` — 
+   is the version we need.
+
+5. Install TypeDB using the versions from the previous command (here we will use typedb-server version 2.15 for 
+   example):
+
+   <!-- test-ignore -->
+   ```bash
+   sudo apt install typedb-server=2.15.0 typedb-console=2.15.0 typedb-bin=2.12.0
+   ```
+
+<div class="note">
+[Note]
+The `typedb-server` and `typedb-console` packages are updated more often than `typedb-bin` that is why their version 
+numbers might differ. By default, APT will look for the exact same version of `typedb-bin` and that will result in an 
+error. To prevent this we use apt show as shown above to display the dependencies of `typedb-server`, take note of 
+the `typedb-bin` version required and invoke an `apt install` command with the specific version of every package.
+</div>
+
+#### Manual
+
+1. Ensure Java 11+ is installed:
+
+   TypeDB supports the [OpenJDK](https://jdk.java.net) and
+   [Oracle JDK](https://www.oracle.com/java/technologies/downloads/).
+
+2. Download TypeDB:
+
+   Download the latest release of [TypeDB](https://vaticle.com/download).
+
+3. Extract archive:
+
+   Create a new directory, extract the contents of the zip file and move them
+   (hereinafter replace `2.15.0` with the version downloaded):
+   
+   <!-- test-ignore -->
+   ```bash
+   mkdir /opt/typedb
+   tar -xzf ~/Downloads/typedb-all-linux-2.15.0.tar.gz
+   mv ~/Downloads/typedb-all-linux-2.15.0 /opt/typedb
+   ```
+   
+4. Add a symlink to the TypeDB executable in the /usr/local/bin directory:
+
+   <!-- test-ignore -->
+   ```bash
+   ln -s /opt/typedb/typedb /usr/local/bin/typedb
+   ```
+
+### Start
+
+Run the following command in a terminal:
+
+<!-- test-ignore -->
+```shell
+typedb server
+```
+
+<div class="note">
+[Note]
+TypeDB will run in the foreground. If the terminal running TypeDB is closed, TypeDB will be shut down. We recommend 
```

Comment:
We could provide guidance on running the server in the background via `nohup typedb server&`, tmux or something else. Not applicable to Windows.

Reply:
Providing a systemd service file might also be useful.

Reply:
But why would we do that? That will only create additional headache for tech-support latter. There are so many ways to do just that but they all have little common with TypeDB. Basically we only want to document features of TypeDB.